README.md

```diff
@@ -45,3 +49,6 @@ for more details.
 [cpplint]: https://github.com/google/styleguide/tree/gh-pages/cpplint
 [emacs]: https://raw.githubusercontent.com/google/styleguide/gh-pages/google-c-style.el
 [xml]: https://google.github.io/styleguide/xmlstyle.html
+[dart]: https://www.dartlang.org/guides/language/effective-dart
+[go]: https://github.com/golang/go/wiki/Style
```

Comment:
Yeah, I saw that. It's just a weirdly named page for a style guide, and I saw that Style linked to it. :) I'll update the link, though.